src/RefractDataStructure.cc

```diff
@@ -723,12 +760,22 @@ namespace drafter {
 
             if (property.node.name.variable.values.size() > 1) {
                 // FIXME: is there example for multiple variables?
-                throw snowcrash::Error("multiple variables in property definition are not allowed", snowcrash::MSONError);
+                throw snowcrash::Error("multiple variables in property definition are not implemented", snowcrash::MSONError);
             }
 
             snowcrash::SourceMap<mson::Literal> sourceMap;
             sourceMap.sourceMap.append(property.sourceMap.name.sourceMap);
 
+            // check if base variable type is StringElement
+            if (!property.node.name.variable.typeDefinition.empty()) {
+                const std::string& type = property.node.name.variable.typeDefinition.typeSpecification.name.symbol.literal;
+                if (!type.empty()) {
+                    if (!refract::TypeQueryVisitor::as<refract::StringElement>(FindRootAncestor(type, GetNamedTypesRegistry()))) {
```

Comment:
I am sorry but I might be confused. Why are we hardcoding this to `StringElement`? All primitive element types are possible. Aren't they?

```
# ID (number)
## Sample
123

# ID2 (ID)
## Sample
456
```